data/field_meta.go

```diff
@@ -0,0 +1,32 @@
+package data
+
+// FieldMeta attaches structural metadata to each frame
+type FieldMeta struct {
+	// StepSize is the interval between each value.
+	// If no values exist in the values array, this indictes mising data for that interval
+	// Only supports non-zero step sizes
+	StepSize float64 `json:"stepSize,omitempty"`
+
+	// Path is a browsable path on the datasource.
+	StepScale string `json:"stepScale,omitempty"`
```

Comment:
sorry, what does this one do?

Reply:
For heatmaps/histograms, the scale could be log2 or log10